pytest/defs.bzl

```diff
@@ -0,0 +1,65 @@
+"""Use pytest to run tests, using a wrapper script to interface with Bazel.
+
+Example:
+
+```starlark
+load("@aspect_rules_py//pytest:defs.bzl", "pytest_test")
+
+pytest_test(
+    name = "test_w_pytest",
+    size = "small",
+    srcs = ["test.py"],
+)
+```
+
+By default, `@pip//pytest` is added to `deps`.
+If sharding is used (when `shard_count > 1`) then `@pip//pytest_shard` is also added.
+To instead provide explicit deps for the pytest library, set `pytest_deps`:
+
+```starlark
+pytest_test(
+    name = "test_w_my_pytest",
+    shard_count = 2,
+    srcs = ["test.py"],
+    pytest_deps = [requirement("pytest"), requirement("pytest-shard"), ...],
+)
+```
+"""
+
+load("//py:defs.bzl", "py_test")
+
+def pytest_test(name, srcs, deps = [], args = [], pytest_deps = None, pip_repo = "pip", **kwargs):
```

Comment:
I don't love the name here, the rest of the rules are `py_*`, representing the language, so technically this should be `py_pytest_test`, even if it is a mouthful.

Reply:
done